header/local/exchange.go

```diff
@@ -43,6 +43,11 @@ func (l *Exchange) GetRangeByHeight(ctx context.Context, origin, amount uint64)
 	return l.store.GetRangeByHeight(ctx, origin, origin+amount)
 }
 
+func (l *Exchange) GetVerifiedRange(ctx context.Context, origin *header.ExtendedHeader, amount uint64,
```

Comment:
Why do we switch from "to" on the interface to "amount" here? These mean different things of course

Reply:
Same Q

Reply:
to be consistent with `GetRangeByHeight`

Reply:
It's from-to on the interface

Reply:
so `GetRangeByHeight` has 😄

Reply:
Our interface defines the param as `to` but impl in exchange is `amount` so we should change interface and all implementations to be consistent.

Reply:
can be done in a subsequent pr

Reply:
No, this PR

Reply:
It is just a var name